pkgs/development/libraries/science/biology/bicpl/default.nix

```diff
@@ -25,6 +25,6 @@ stdenv.mkDerivation rec {
     description = "Brain Imaging Centre programming library";
     maintainers = with maintainers; [ bcdarwin ];
     platforms = platforms.unix;
-    license   = licenses.free;
+    license   = licenses.hpndUc;
```

Comment:
vendored `Images/ppm/COPYING` contains gpl3plus. does that matter?

Reply:
Yes they should be added